### PR TITLE
[tenable_sc] Add User-Agent and batch_size config

### PR DESCRIPTION
**Pull request by darshan-elastic**
## What does this PR do?

- Add visualizations to search by host and vulnerability dashboard
- Added the custom_hash for finding uniqueness in Assets
- Added configurable page parameter
- Added specific user-agent for httpjson requests for Tenable.sc

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/tenable_sc directory.

Run the following command to run tests.
`elastic-package test`

## Related issues

- Relates #2348

## Screenshots
image
image
image
image
image
image
image

### CODE REVIEW

File: packages/tenable_sc/changelog.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # newer versions go on top
-- version: 1.0.0
+- version: "1.1.0"
+  changes:
+    - description: Add custom User-Agent. Added configurable response size. Added filter in vulnerability dashboard to filter hostname and vulnerability cve id. Added unique identifier to asset.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/xxxx
```
Comment (efd6) [inline packages/tenable_sc/changelog.yml:6]: s/xxxx/2718/

File: packages/tenable_sc/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: tenable_sc
 title: Tenable.sc
-version: 1.0.0
+version: 1.1.0
```
Comment (andrewkroh) [inline packages/tenable_sc/manifest.yml:4]: Please add a comment here indicating that the version must be updated in the pipeline as well.

Until https://github.com/elastic/kibana/issues/121310 is implemented we will have to manually sync these.

File: packages/tenable_sc/data_stream/asset/agent/stream/httpjson.yml.hbs [hunk 3]
```diff
@@ -10,6 +10,11 @@ request.proxy_url: {{proxy_url}}
 request.ssl: {{ssl}}
 {{/if}}
 request.transforms:
+  - delete:
+      target: header.User-Agent
+  - set:
+      target: header.User-Agent
```
Comment (andrewkroh) [inline packages/tenable_sc/data_stream/asset/agent/stream/httpjson.yml.hbs:16]:
```suggestion
      # Follow Tenable's format: https://developer.tenable.com/docs/user-agent-header
      # NOTE: The "Build" version must be kept in sync with this package's version. 
      target: header.User-Agent
```

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**